classes/class-wc-connect-taxjar-integration.php

```diff
@@ -52,6 +56,33 @@ public function init() {
 	}
 
 	/**
+	 * Add our "automated taxes" setting to the core group.
+	 *
+	 * @param array $tax_settings WooCommerce Tax Settings
+	 *
+	 * @return array
+	 */
+	public function add_tax_settings( $tax_settings ) {
+		$automated_taxes = array(
+			'title'    => __( 'Automated taxes', 'woocommerce-services' ),
+			'id'       => self::OPTION_NAME, // TODO: save in `wc_connect_options`?
+			'desc_tip' => __( 'Automate your sales tax calculations with WooCommerce Services.', 'woocommerce-services' ),
+			'default'  => 'no',
+			'type'     => 'select',
```

Comment:
Wouldn't this be better as a checkbox?

Reply:
Maybe - it was designed as a select, so that's what I went with. We can revisit this later. Bigger 🐟  to 🔥 .